### ISSUE TITLE
Use online services such as weblate or transifex for localization rather than raw GitHub commits

### ISSUE THREAD

**Issue by kleenleen** [pop-os/cosmic-epoch#1362]
I think you are wasting lots of energy and potential regarding your localization process currently.

First, your devs have to manually manage fluent commits one by one.

Second, only a few contributors with GitHub knowledge can contribute, and it's not a majority of people.

Third, it makes the progress very uneven and hard to follow.

By using specialized platforms, you could better manage the translations progress, have better and more contributions, while making is easier for you.

That would be a win-win change.

**Comment by leviport** [comment]
Duplicate of https://github.com/pop-os/cosmic-epoch/issues/232
